src/main/resources/templates/snowflake/shared/create-table.ssp

```diff
@@ -13,13 +13,13 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 --%>
-#import(io.phdata.streamliner.configuration.Snowflake)
-<%@ val configuration: io.phdata.streamliner.configuration.Configuration %>
-<%@ val table: io.phdata.streamliner.configuration.SnowflakeTable %>
+#import(io.phdata.streamliner.schemadefiner.model.Snowflake)
+<%@ val configuration: io.phdata.streamliner.schemadefiner.model.Configuration %>
+<%@ val table: io.phdata.streamliner.schemadefiner.model.SnowflakeTable %>
 <%@ val typeMapping: Map[String, Map[String, String]]%>
 #{
-    val destination = configuration.destination.asInstanceOf[Snowflake]
+    val destination = configuration.getDestination.asInstanceOf[Snowflake]
 }#
-CREATE TABLE IF NOT EXISTS ${destination.stagingDatabase.name}.${destination.stagingDatabase.schema}.${table.destinationName} (
+CREATE TABLE IF NOT EXISTS ${destination.getStagingDatabase.getName}.${destination.getStagingDatabase.getSchema}.${table.getDestinationName} (
```

Comment:
Is it possible to make this value public so we don't have to make this change?

Reply:
updated